src/test/java/teammates/test/cases/browsertests/InstructorCourseEnrollPageUiTest.java

```diff
@@ -134,8 +134,8 @@ private void testEnrollAction() throws Exception {
             .withUserId(testData.instructors.get("CCEnrollUiT.teammates.test").googleId)
             .withCourseId(courseId);
         detailsPage = loginAdminToPage(coursesPageUrl, InstructorCoursesDetailsPage.class);
-        assertEquals(3, detailsPage.getStudentCountForCourse("CCEnrollUiT.CS2104"));
-
+        assertEquals(3, detailsPage.getStudentCountForCourse());
+      
```

Comment:
@sponde25 see the tag **s.ToMerge** it means its approved and yet to be merged. So your job is done.